packages/react-native/src/BacktraceClient.ts

```diff
@@ -42,7 +42,7 @@ export class BacktraceClient extends BacktraceCoreClient<BacktraceConfiguration>
             },
             requestHandler: new BacktraceBrowserRequestHandler(clientSetup.options),
             debugIdMapProvider: new VariableDebugIdMapProvider(global as DebugIdContainer),
-            stackTraceConverter: new ReactStackTraceConverter(new V8StackTraceConverter()),
+            stackTraceConverter: new ReactStackTraceConverter(new V8StackTraceConverter('address-at')),
```

Comment:
should it be `address at`?

Reply:
yeah, it should be `address at`. Updated